### PR TITLE
add sample option

### PR DESCRIPTION
**Pull request by roger-liu-bina**
- [ ] [RES-XXXX](https://binatechnologies.atlassian.net/browse/RES-XXXX)
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: ADD HERE IF ANY
- [ ] Release note (if needed)
- [x] No code copied from outside Bina 
- [ ] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

Briefly describe the changes and reasons for the changes.

### CODE REVIEW

File: generate_small_test_ref.py [hunk 1]
```diff
@@ -68,33 +70,44 @@ def uint(value):
...
-          logger.error("Failed to process INFO %s" % str(record.info))
-          logger.error("more warnings will be suppressed")
-          info_warned = True
-      vcfrecord = vcf.model._Record(record.contig, record.pos + 1, record.id, record.ref, [record.alt,], record.qual, record.filter, info, fmt, sample_indexes)
-      if len(record):
-        vcfrecord.samples = vcf_template_reader._parse_samples([record[0]], "GT", vcfrecord)
-      vcf_writer.write_record(vcfrecord)
+      if not record:
+          continue
+      if record.POS <= region.start + args.flank or record.POS + len(record.CHROM) + args.flank - 1 >= region.end: continue
```
Comment (marghoob) [inline generate_small_test_ref.py]: `len(record.CHROM)` seems incorrect. Should be something like `len(record.REF)`.

File: generate_small_test_ref.py [hunk 2]
```diff
@@ -90,8 +90,6 @@ def uint(value):
     except ValueError: logger.error("Failed to retrieve %s from %s" % (str(region).strip(), invcf))
     if records is None: continue
     for record in records:
-      if not record:
```
Comment (yunfeiguo) [inline generate_small_test_ref.py]: Hi @roger-liu-bina, I wonder why these 2 lines are considered redundant? Thanks.

Comment (roger-liu-bina) [inline generate_small_test_ref.py]: I assume all the record will not be empty. Right? @marghoob

Comment (marghoob) [inline generate_small_test_ref.py]: Yes, @roger-liu-bina it should not be empty. `not record` was not there earlier and it worked. Roger added to his pull request and later removed it.

**Comment by marghoob** [comment]
@yunfeiguo and @roger-liu-bina can this be merged?

**Comment by roger-liu-bina** [comment]
@marghoob yes. Please merge.

**Pull request merged**